.storybook/locale.ts

```diff
@@ -220,6 +220,8 @@ let locale: Record<string, string> = {
   rewardsRestoreText3: 'Restoring with a recovery key will replace your current wallet. So make sure you empty or back up your current wallet before restoring.',
   rewardsRestoreText4: 'Enter your recovery key or',
   rewardsSummary: 'Rewards Summary',
+  rewardsViewQRCodeText1: 'View QR Code',
+  rewardsViewQRCodeText2: 'for iOS Rewards users transferring BAT.',
```

Comment:
There are a couple of strings that were used in `qrBox.tsx` ("qrBoxText" and "qrBoxButton"). Do we still need those?

Reply:
Nope, removed - thanks!